modules/ROOT/pages/workday/workday-connector-studio.adoc

```diff
@@ -100,11 +98,15 @@ org.mule.tooling.client.api.exception.TimeoutException: Mule Agent REST service
 
 === Solution
 
-When one of these conditions occurs, increase the values for `Default Connection Timeout` and `Default Read Timeout` in Studio at Preferences > Anypoint Studio > Tooling.  Double the values, apply, and restart Studio.
+When one of these conditions occurs:
+
+. In Anypoint Studio, navigate to *Preferences > Anypoint Studio > Tooling*.  
+. Double the values for `Default Connection Timeout` and `Default Read Timeout`. 
+. Click *Apply*, and restart Studio.
 
 == Next Step
 
-After configuring the Workday Connector for use in Studio, see the xref:workday/workday-connector-examples.adoc[Examples] to experiment with the connector.
+After configuring the Workday Connector in Studio, see the xref:workday/workday-connector-examples.adoc[Examples] to experiment with the connector.
```

Comment:
```suggestion
After configuring Workday Connector in Studio, see the xref:workday/workday-connector-examples.adoc[Examples] to experiment with the connector.
```
Can we say something more specific about "experiment with the connector"? Now that I know what those experiments are, maybe "...to review some examples of using the connector to perform tasks like <name the two example tasks>."?